pinot-core/src/main/java/com/linkedin/pinot/core/operator/MCombineGroupByOperator.java

```diff
@@ -49,12 +49,14 @@
 public class MCombineGroupByOperator extends BaseOperator {
```

Comment:
do we have multiple combine operators? i.e. group by, aggregation only, selection?. There is a lot of boilerplate code that can be extracted to one class